spaces/S000135/properties/P000132.md

```diff
@@ -0,0 +1,18 @@
+---
+space: S000135
+property: P000132
+value: true
+refs:
+- doi: 10.1007/978-1-4612-6290-9_6
+  name: Counterexamples in Topology
+---
+
+Since $X\setminus\{(0,0)\}$ is homeomorphic to the disjoint union of continuum many copies of the real line,
+every open set in $X$ missing the origin is $F_\sigma$.
```

Comment:
```suggestion
Since $X\setminus\{(0,0)\}$ is homeomorphic to the disjoint union of copies of the real line
and {S25|P132}, every open set in $X$ missing the origin is an $F_\sigma$.
```